workers_kv.go

```diff
@@ -83,18 +84,53 @@ func (api *API) CreateWorkersKVNamespace(ctx context.Context, req *WorkersKVName
 //
 // API reference: https://api.cloudflare.com/#workers-kv-namespace-list-namespaces
 func (api *API) ListWorkersKVNamespaces(ctx context.Context) (ListWorkersKVNamespacesResponse, error) {
-	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces", api.AccountID)
+	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces?per_page=100", api.AccountID)
 	res, err := api.makeRequestContext(ctx, http.MethodGet, uri, nil)
 	if err != nil {
 		return ListWorkersKVNamespacesResponse{}, errors.Wrap(err, errMakeRequestError)
 	}
 
-	result := ListWorkersKVNamespacesResponse{}
-	if err := json.Unmarshal(res, &result); err != nil {
-		return result, errors.Wrap(err, errUnmarshalError)
+	overallResult := ListWorkersKVNamespacesResponse{}
```

Comment:
This function returns a `ListWorkersKVNamespacesResponse` as opposed to, for example, `[]WorkersKVNamespace`. This makes it awkward to return the full list. In the current changeset, I'm hijacking the first `ListWorkersKVNamespacesResponse` and augmenting its `Result` list, but leaving `ResultInfo` and `Response` alone.

Would it make more sense to move the paging into a separate method which returns a `[]WorkersKVNamespace`.

Reply:
Yeah, I would expect this to match the other methods like it and return `[]WorkersKVNamespace`. I'm not sure if we have use for the `ListWorkersKVNamespacesResponse` as is.

Reply:
Would it make sense to have that breaking change in this PR?